WcaOnRails/app/views/shared/_associated_events_picker.html.erb

```diff
@@ -23,7 +23,7 @@ Note: form_builder.object is the object having associated events.
 
 <div class="form-group <%= "has-error" unless errors.empty? %>">
   <%= label_tag events_association_name, class: "associated-events-label" do %>
-    <%= label %>
+    <%= label %> (<span class="events-selected-count"></span> selected)
```

Comment:
Yeah, but I have pluralization on mind, in Polish it would be:

- Konkurencje (0 wybranych)
- Konkurencje (1 wybrana)
- Konkurencje (2 wybrane)
- Konkurencje (14 wybranych)

Whereas following my suggestion or something similar, it would be

- Konkurencje (liczba wybranych: <any number fits here>)